pkg/controller/migmigration/restore.go

```diff
@@ -224,6 +224,30 @@ func getPodVolumeRestoresProgress(pvrList *velero.PodVolumeRestoreList) (progres
 	return
 }
 
+// Returns the restore progress statistics
+func getRestoreStats(restore *velero.Restore) (itemsRestored int, totalItems int) {
+	if restore.Status.Progress == nil {
```

Comment:
Please add a nil check for restore object itself

Reply:
Done, updated the PR.